website/docs/r/lex_slot_type.html.markdown

```diff
@@ -0,0 +1,125 @@
+---
+layout: "aws"
+page_title: "AWS: aws_lex_slot_type"
+sidebar_current: "docs-aws-resource-lex-slot-type"
+description: |-
+  Provides an Amazon Lex Slot Type resource.
+---
+
+# Resource: aws_lex_slot_type
+
+Provides an Amazon Lex Slot Type resource. For more information see
+[Amazon Lex: How It Works](https://docs.aws.amazon.com/lex/latest/dg/how-it-works.html)
+
+## Example Usage
+
+```hcl
+resource "aws_lex_slot_type" "flower_types" {
+  create_version = false
+  description    = "Types of flowers to order"
+
+  enumeration_value {
+    synonyms = [
+      "Lirium",
+      "Martagon",
+    ]
+
+    value = "lilies"
+  }
+
+  enumeration_value {
+    synonyms = [
+      "Eduardoregelia",
+      "Podonix",
+    ]
+
+    value = "tulips"
+  }
+
+  name                     = "FlowerTypes"
+  value_selection_strategy = "ORIGINAL_VALUE"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `create_version` _(Optional)_:
+
+  Determines if a new slot type version is created when the initial resource is created and on each
+  update. Defaults to true.
+
+* `description` _(Optional)_:
+
+	A description of the slot type.
+
+* `enumeration_value` _(**Required**)_:
+
+	A list of EnumerationValue objects that defines the values that the slot type can take. Each
+	value can have a list of synonyms, which are additional values that help train the machine
+	learning model about the values that it resolves for a slot. Attributes are documented under
+  [enumeration_value](#enumeration_value-1).
+
+* `name` _(**Required**)_:
```

Comment:
These don't need to be bold and in italics. The Required/Optional keywords do not get styled in anyway other than being wrapped in parentheses. The description for the argument should be inline.
 
```suggestion
* `name` (Required): The name of the slot type. The name is not case sensitive.
```